docs/faq.md

```diff
@@ -20,15 +20,42 @@ Skin can be simulated using the [AdnSkin](maya/skin) deformer. This deformer req
 
 Similarly to a skin simulation setup, the fascia can be simulated using [AdnSkin](maya/skin) deformer as well. In this case, it is recommended to use values of *Rest Length Multiplier* lower than 1. This deformer also requires one mesh (a single target, i.e. the muscle geometries combined) or multiple meshes (multiple targets, i.e. the muscle geometries separated) to drive the simulation and a mesh to apply the deformer to (fascia geometry).
 
-## Can I simulate muscles, fascia and skin all coupled?
+## How can I generate the fascia geometry?
 
-Yes, you can simulate muscles, fascia and skin following these steps:
+In order to obtain a plausible fascia geometry, we can use two techniques that require the muscle geometries and the skin geometry as inputs.
+
+The easiest option is to use the shrinkwrap deformer provided by Maya. This solution works better for low poly geometries.
+
+- Create a copy of the skin geometry that will become your fascia.
+- Select the combined geometry of all muscles, then select the copy of the skin geometry.
+- Apply the shrinkwrap deformer and adjust its settings until the desired fascia geometry is achieved.
+- Delete the history of the fascia geometry.
+
+For high-end fascia simulations, a better option would be to create the fascia geometry using a pressured cloth simulation. nCloth can be used to achieve this:
+
+- Create a copy of the skin geometry that will become your fascia.
+- Select the fascia geometry and click nCloth > Create nCloth.
+- Select the muscle geometries and click on nCloth > Create Passive Collider.
+- Select the nucleus node and set the gravity to 0.
+- Select the nCloth node and keyframe the pressure attribute to 0 at the first frame of the scene.
+- Advance some frames (e.g. ten frames) and keyframe the pressure to -1.
+- Select the fascia geometry and click nCloth > Create New nCache > nObject.
+- Once the simulation finishes, choose the fascia that looks best within the simulated frame range.
```

Comment:
I think we have to delete the history too in order to get a clean static geometry. Could you please double check?

Reply:
Correct! I would also advise something for later. As we require the fat layer to have the same topology as the skin layer it is importand that the fascia is subdivided the same way as the fat layer (skin duplicated essentially). And maybe we can suggest to add some DeltaMush to smoothen even firther the fascia? Maybe that can be optional. CC: @carlosmonteagudoinbibo

Reply:
Added a step for deleting the history. Also added an optional step for increasing the subdivisions on the fascia if more resolution is needed. Let me know if we should add a step related to applying DeltaMush. @carlosmonteagudoinbibo.

Reply:
I see the benefits of smoothing, but let's check with Francesco. If it is something CFX artist use to do, yes, add it. Otherwise let's not add it.

Reply:
CC: @francescogioiainbibo